src/turbo-stream.ts

```diff
@@ -88,8 +91,10 @@ async function decodeDeferred(
     const line = read.value;
     switch (line[0]) {
       case TYPE_PROMISE: {
+        const isError = line[1] === TYPE_ERROR;
```

Comment:
The `case TYPE_ERROR:` was extremely similar, and falls into the `TYPE_PROMISE` case now that I slightly changed the format, so I could delete that second `case` with a few minor adjustments.